common/djangoapps/terrain/stubs/http.py

```diff
@@ -217,6 +218,8 @@ def _format_msg(self, format_str, *args):
         `format_str` is a string with old-style Python format escaping;
         `args` is an array of values to fill into the string.
         """
+        if not args:
+            format_str = urllib.unquote(format_str)
```

Comment:
This was to fix args error, when we pass encoded urls specifically course id contains `%` sign. This will break at line 226.